src/averageAges.js

```diff
@@ -15,11 +15,13 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filterMen = !century
+    ? people.filter(person => person.sex === 'm')
+    : people.filter(person => (
+      person.sex === 'm' && Math.ceil(person.died / 100) === century));
+
+  return filterMen.reduce((sum, per) => (
```

Comment:
reduce take a callback with next arguments:
(previousValue, currentValue, index, array) where 
`previousValue` - is the result of your previous action, like if you are adding all values in the array step by step it'll be written inside this parameter -> reduce(callback, `0`). Also if you didn't write start value -> reduce(callback) and passed only callback the first value of the array will be written inside the `previousValue` the next value will be written inside the `currentValue`.

`currentValue` - so this parameter of callback takes a current item of the array.

I thought you need this description because your second parameter of callback inside the reduce is named as `pre` - previousValue I think)